sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -46,21 +46,25 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder, Action<
                         "EventProcessorOptions:InvokeProcessorAfterReceiveTimeout",
                         options.InvokeProcessorAfterReceiveTimeout);
 
-                    options.EventProcessorOptions.TrackLastEnqueuedEventProperties = section.GetValue(
+                    options.TrackLastEnqueuedEventProperties = section.GetValue(
                         "EventProcessorOptions:EnableReceiverRuntimeMetric",
-                        options.EventProcessorOptions.TrackLastEnqueuedEventProperties);
+                        options.TrackLastEnqueuedEventProperties);
 
                     options.MaxBatchSize = section.GetValue(
                         "EventProcessorOptions:MaxBatchSize",
                         options.MaxBatchSize);
 
+                    options.PrefetchCount = section.GetValue(
+                        "EventProcessorOptions:PrefetchCount",
+                        options.PrefetchCount);
+
                     var receiveTimeout = section.GetValue<TimeSpan?>(
                         "EventProcessorOptions:ReceiveTimeout",
                         null);
 
                     if (receiveTimeout != null)
                     {
```

Comment:
As far as I can tell, this style is equivalent to passing the options as the default, so we can probably be consistent and always pass the options as the default value.

Reply:
It is valid to pass `null` for this;  that indicates that no maximum wait time is desired.   I don't think we want to assume a default interval for it unless you've got an alternate way to express "only call me if there is an event"

Reply:
> I don't think we want to assume a default interval for it unless you've got an alternate way to express "only call me if there is an event"

There is the InvokeProcessorAfterReceiveTimeout property.